components/src/dynamo/vllm/tests/test_vllm_worker_factory.py

```
@@ -47,6 +48,80 @@ def _make_config(**overrides) -> Mock:
+    endpoint.connection_id.return_value = 7
+    endpoint.serve_endpoint = AsyncMock(side_effect=serve)
+    runtime = Mock()
+    runtime.endpoint.return_value = endpoint
+
+    engine_client = Mock()
+    vllm_config = Mock()
+    engine_tuple: EngineSetupResult = (
+        engine_client,
+        vllm_config,
+        Mock(),
+        "/tmp/prom",
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/prom"

(S108)